docker-compose-dist.override.yml

```diff
@@ -1,65 +1,35 @@
-version: "3"
+version: "3.4"
 
 services:
```

Comment:
Need to comment out this line as well, to avoid:

```bash
$ docker-compose up requirements mfr_requirements wb_requirements
ERROR: In file './docker-compose.override.yml', service must be a mapping, not a NoneType.
```